plugin-server/src/types.ts

```diff
@@ -219,6 +219,9 @@ export interface PluginsServerConfig {
     SESSION_RECORDING_PARTITION_REVOKE_OPTIMIZATION: boolean
     SESSION_RECORDING_PARALLEL_CONSUMPTION: boolean
     SESSION_RECORDING_CONSOLE_LOGS_INGESTION_ENABLED: boolean
+    // a single partition which will output many more log messages to the console
+    // useful when that partition is lagging unexpectedly
+    SESSION_RECORDING_DEBUG_PARTITION: number | undefined
```

Comment:
Only tweak I would like is for this to be a string that we split by "," to have multiple debuggable partitions.

Not blocking though

Reply:
Yep, I think I'll keep fiddling with this and bear that in mind... lack of visibility is so frustrating!